packages/stream_chat_flutter/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 🔄 Changed
 
 - Animation curves changed from default `Curves.linear` to `Curves.easeOut` and `Curves.easeIn` for attachment controls.
+- Removed default padding in `DateDivider` in `MessageListView`
```

Comment:
Just needs a full stop here. The rest LGTM

Reply:
It's already merged :|